### PR TITLE
Add navigation bar. Change color scheme.

### PR DESCRIPTION
**Pull request by laura-georgescu**

### CODE REVIEW

File: laura/portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -6,27 +6,34 @@
...
 
     <div class="row">
         <div class="column">
-            <p> Hello, there! I am Laura, I'm 19yo and I live in Bucharest, Romania.
-            Since 2019, I study Computer Science at University College, Oxford (the oldest college in the university, founded in 13th century).</p>
+            <p> Hello, there! I am Laura, I'm 19yo and I am from Bucharest, Romania.</p>
+            <p>Since 2019, I study Computer Science at University College, Oxford (the oldest college in the university, founded in 13th century).</p>
             
-            <img src="images/univ.jpg" height="300"/>  
+            <img src="images/univ.jpg" height="400" align="center"/>
```
Comment (stofee) [inline laura/portfolio/src/main/webapp/index.html:24]: maybe use % so it looks good on all screen size?

File: laura/portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -6,27 +6,34 @@
...
         
         <div class="column">
+            <p> In my spare time I enjoy sports -fencing, swimming and skiing in particular-, going out with friends, cooking new recipes and traveling.</p>
+            <p> Check me out on  <a href="https://linkedin.com/in/laura-ioana-georgescu" style="color:white;">LinkedIn</a> and 
+            <a href="https://codeforces.com/profile/laurageorgescu" style="color:white;">Codeforces</a>. </p>
             <p> This is my portofolio for the 2020 STEP internship at Google.</p>
-            <p> In my spare time I enjoy sports (fencing and skiing in particular), going out with friends, cooking new recipes and traveling.</p>
-       
-            <img src="images/me.JPG" height = "300"/>          
+            <img src="images/me.JPG" height = "350" align="center"/>
```
Comment (stofee) [inline laura/portfolio/src/main/webapp/index.html:32]: same as above

File: laura/portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -9,37 +9,45 @@
 }
 
 body {
-  background-color: ghostwhite;
+  background-color: #2F3131;
```
Comment (stofee) [inline laura/portfolio/src/main/webapp/style.css:12]: add a comment on what this vlaue is... also you use it multiple times, extracting it to a const might helps a potential refactoring later.

File: laura/portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (laura-georgescu) [reply laura/portfolio/src/main/webapp/index.html]: solved in new commit.

File: laura/portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (laura-georgescu) [reply laura/portfolio/src/main/webapp/index.html]: same

File: laura/portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -6,30 +6,35 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  
+    <ul>
+        <li><a href="index.html" class="navbar">Home</a>
+            <a href="mood.html" class="navbar">Mood generator</a></li>
+    </ul>
+
```
Comment (lanseg) [inline laura/portfolio/src/main/webapp/index.html:14]: Navbar should be inside "body".
Also, there is a special tag "<nav>" for elements like this: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

File: laura/portfolio/src/main/webapp/mood.html [hunk 5]
```diff
@@ -6,6 +6,12 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
+    <ul>
+        <li><a href="index.html" class="navbar">Home</a>
+            <a href="mood.html" class="navbar">Mood generator</a></li>
+    </ul>
+
```
Comment (lanseg) [inline laura/portfolio/src/main/webapp/mood.html:14]: Same as for the main page, navbar should be inside <body> and you can have look at "<nav>" tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

File: laura/portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (laura-georgescu) [reply laura/portfolio/src/main/webapp/index.html]: done in new commit.

File: laura/portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -6,13 +6,14 @@
...
-        <li><a href="index.html" class="navbar">Home</a>
-            <a href="mood.html" class="navbar">Mood generator</a></li>
-    </ul>
-
   <body>
+    <nav>
+        <ul>
+            <li><a href="index.html" class="navbar">Home</a>
+                <a href="mood.html" class="navbar">Mood generator</a></li>
+        </ul>
```
Comment (lanseg) [inline laura/portfolio/src/main/webapp/index.html:14]: It's better to have each navigation element inside it's own li.
Simple example could be found here: https://www.w3schools.com/css/css_navbar.asp

**Pull request merged**